packages/svelte-ux/src/routes/docs/components/DateRange/+page.svelte

```diff
@@ -57,8 +60,8 @@
 <Preview>
   <DateRange
     periodTypes={[PeriodType.Day, PeriodType.Month]}
-    getPeriodTypePresets={(periodType) => {
-      const { start, end, add } = getDateFuncsByPeriodType(periodType);
+    getPeriodTypePresets={(settings, periodType) => {
```

Comment:
This should be `$localeSettings`

Reply:
I think that it's name of args, not `$localeSettings`, but I'm maybe wrong